_includes/footer.html

```diff
@@ -14,7 +14,7 @@
           </div>
           <div class="hidden-sm hidden-xs col-md-2 innovation-footer-image-wrapper">
              <a href="http://yourfuture.asu.edu/rankings" target="_blank" id="best-colleges-us-news-bage-icon">
-              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme{{site.base_path}}/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
+              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
```

Comment:
We should have this footer image come from this theme rather than the sustainability site.
`{{site.base_path}}/assets/bootstrap-asu/build/img/footer/best-colleges-us-news-badge.png`